Makefile

```diff
@@ -55,13 +60,18 @@ onos-config-docker: # @HELP build onos-config Docker image
     	--build-arg ONOS_BUILD_VERSION=${ONOS_BUILD_VERSION} \
 		-t onosproject/onos-config:${ONOS_CONFIG_VERSION}
 
+onos-config-docker-debug: # @HELP build onos-config Docker debug image
+	docker build . -f build/onos-config-debug/Dockerfile \
+		--build-arg ONOS_BUILD_VERSION=${ONOS_BUILD_VERSION} \
+		-t onosproject/onos-config:${ONOS_CONFIG_DEBUG_VERSION}
+
 onos-cli-docker: # @HELP build onos-cli Docker image
 	docker build . -f build/onos-cli/Dockerfile \
         --build-arg ONOS_BUILD_VERSION=${ONOS_BUILD_VERSION} \
         -t onosproject/onos-cli:${ONOS_CONFIG_VERSION}
 
 images: # @HELP build all Docker images
-images: onos-config-docker onos-cli-docker
+images: onos-config-docker onos-config-docker-debug onos-cli-docker
```

Comment:
I wonder if it would be better to have an "images debug" other wise if I go "make run-docker" it will have to go off and build, vet, lint and test all 3 images everytime!

Reply:
Do we even need the CLI image for `run-docker`? Doesn't it make sense to just run `onos-config-docker` in `run-docker`?

Reply:
That's a question for Thomas - I presume he had some reason for putting it in there